### DISCUSSION TITLE
Zone Bypass status

### DISCUSSION THREAD

**Discussion by ChrisPotDev** [Dilbert66/esphome-dsckeybus#32] [Ideas]
Any chance of having zone bypass status available?
I have a scenario where I need to bypass a number of zones from time to time and it would be useful to have an Indication of the zones that are bypassed.

**Comment by Dilbert66** [comment]
Ability to view and bypass zones is part of some changes I'm adding to support full keypad emulation. I don't have a timeline yet.

**Reply by ChrisPotDev** [reply]
Thank you kind sir

**Reply by bgardner46** [reply]
I know this is quite a bit later, but is bypass status available yet. By the way, this is my favorite ESPHome implementation. It works great and bypassing zones is extremely simple. Thanks for your efforts.
Bob

**Comment by Dilbert66** [comment]
Bypassed, alarmed, open zones are listed in the "zs" zone status sensor.  Bypassed zones will be shown as "BY:xx" where xx is the zone number.

**Comment by bgardner46** [comment]
I did not realize this. Thanks so much for the info.
